pytorch_lightning/strategies/deepspeed.py

```diff
@@ -321,6 +337,83 @@ def __init__(
         self.hysteresis = hysteresis
         self.min_loss_scale = min_loss_scale
 
+    @property
```

Comment:
Let's be consistent and order property as DDP. This would make debugging and readability simpler.